wallet/src/money.rs

```diff
@@ -18,6 +18,57 @@ use tuxedo_core::{
     verifier::Sr25519Signature,
 };
 
+use std::{thread::sleep, time::Duration};
+
+/// The default number of coins to be minted.
+const DEFAULT_NUM_OF_COINS: u128 = 100;
```

Comment:
All of the default handling should happen at the CLI level. By the time we get here, we expect data.